_includes/social-share/index.html

```diff
@@ -1,35 +1,48 @@
-<ul>
+<section class="fixed-l top-2-l dn-l nl6-l" id="social-share-container">
+  <div class="ttu f6 dn-l tc pt4 mid-gray silver b">Enjoyed the article? Share it!</div>
+  <ul class="list pl0 flex justify-center flex-column-l">
     <!-- Share on Facebook -->
     <li>
-        <a href="http://www.facebook.com/sharer.php?u={{ site.url }}{{ page.url }}"
-           target="_blank" title="Share on Facebook" rel="nofollow"
-           onclick="return shareOnFacebook()"><div class="w2">{% include social-share/facebook.svg %}</div></a>
```

Comment:
Please remove all these changes so that my PR will not conflict but also because this is not working as expected. I will try and consolidate the JS for some SM profiles but please note that Twitter must have a different method/function as discussed yesterday.